setup.cfg

```diff
@@ -1,3 +1,16 @@
 [flake8]
 ignore = E302, W503, E501, E741, E203, F405
 max-line-length = 88
+
+[codespell]
+skip = *.pyc,*.gif,*.png,*.jpg,*.ply, ./bibliography.bib,*.ipynb
+ignore-words-list = lod,byteorder,flem,parm,doubleclick,revered,inout,fro,nd,sies,hist,ans
+quiet-level = 3
+
+[tool.isort]
```

Comment:
Do we need `isort` when using `ruff`? Based on my understanding, `ruff` replaces `isort`.

Reply:
@dafeda indeed I was not aware that ruff now includes isort. I made the changes and squashed my commits.